### DISCUSSION TITLE
Community meeting on 24th Oct

### DISCUSSION THREAD

**Discussion by pavithraes** [nebari-dev/nebari#2085] [Community meetings]
We'll have a community meeting in a few minutes, please add discussion topics and updates to the meeting notes!

🕒 Meeting date & time: 2-3 pm GMT

💻 Meeting room: [meet.google.com/rio-ngrj-kyh](https://meet.google.com/rio-ngrj-kyh)

📝 Meeting notes: https://hackmd.io/@nebari/HJIbdHSMT

🗓️ Meeting calendar: [nebari.dev/docs/community#community-meetings](https://www.nebari.dev/docs/community#community-meetings)

**Comment by davidchapuis** [comment]
Hi, is there a meeting today?
Interested in joining

**Reply by Adam-D-Lewis** [reply]
Yes, there is a meeting right now at the google meet link above.

**Reply by davidchapuis** [reply]
Nice, I just asked to join the meeting

**Reply by davidchapuis** [reply]
Nobody let me in

**Reply by Adam-D-Lewis** [reply]
Sorry about that @davidchapuis   I wasn't in that meeting, but happened to see your comment and responded.  The attendees may not have seen your attempt to join.  Feel free to open an issue if there's something you wanted to bring up, or please join the next community meeting.
